src/platform/silabs/wifi/WifiInterface.cpp

```diff
@@ -31,6 +31,9 @@ using namespace chip::DeviceLayer;
 
 #define CONVERT_SEC_TO_MS(TimeInS) (TimeInS * 1000)
 
+// TODO: We shouldn't need to have access to a global variable in the interface here
+extern WfxRsi_t wfx_rsi;
```

Comment:
This variable should have access utility functions with thread safety mechanisms

Reply:
As discussed, will be in a follow up